myst_nbs/gaussian_processes/GP-Kron.myst.md

```diff
@@ -241,11 +275,33 @@ axs = axs.ravel()
 for i, ax in enumerate(axs):
     ax.axis("off")
     ax.scatter(X[:, 0], X[:, 1], s=20, c=y, marker="o", norm=norm, cmap=cmap)
-    ax.scatter(Xnew[:, 0], Xnew[:, 1], s=20, c=ppc["fnew"][i], marker="s", norm=norm, cmap=cmap)
+    ax.scatter(
+        Xnew[:, 0],
+        Xnew[:, 1],
+        s=20,
+        c=ppc.posterior_predictive["fnew"].sel(chain=0)[i],
+        marker="s",
+        norm=norm,
+        cmap=cmap,
+    )
     ax.set_title(f"Sample {i+1}", fontsize=24)
 ```
 
+## Authors
+* Authored by [Bill Engels](https://github.com/bwengals), 2018
+* Updated by [Chris Fonnesbeck](https://github.com/fonnesbeck), [Raul-ing Average](https://github.com/CloudChaoszero), [Christopher Krapu](https://github.com/ckrapu), [Hector Munoz](https://github.com/hectormz), and [Danh Phan](https://github.com/danhphan), 2020-2022
+
++++
+
+## References
+:::{bibliography}
+:filter: docname in docnames
+:::
```

Comment:
Same comment as in the other notebooks, there are no references using the cite role (as shown in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#references) so this shoud be omited. The watermark section title is missing below though